server/content.js

```diff
@@ -23,10 +24,15 @@ Meteor.methods({
     }
 
     // TODO: Check more permissions?
-
     let addedCount = 0;
     const latestContent = Content.documents.findOne({contentKey: args.contentKey}, {sort: {version: -1}, fields: {version: 1}});
-
+    const document = Document.documents.findOne({contentKey: args.contentKey});
+    if (document.isPublished()) {
+      // If the document is published we immediately discard this new step
+      // In the future, we should check if the content that they're trying
+      // to add is a comment, to allow that.
+      return 0;
```

Comment:
But this prevents then commenting, no?

Reply:
Right now, yes. Comments won't be stored. When PR #33 is merged, we should add comments as an exception, to allow them to be stored.

Reply:
Return `addedCount` here (which is 0).